be/src/olap/aggregate_func.h

```diff
@@ -449,15 +449,22 @@ struct AggregateFuncTraits<OLAP_FIELD_AGGREGATION_REPLACE_IF_NOT_NULL, OLAP_FIEL
 template <>
 struct AggregateFuncTraits<OLAP_FIELD_AGGREGATION_HLL_UNION, OLAP_FIELD_TYPE_HLL> {
     static void init(RowCursorCell* dst, const char* src, bool src_null, MemPool* mem_pool, ObjectPool* agg_pool) {
-        DCHECK_EQ(src_null, false);
         dst->set_not_null();
 
         auto* src_slice = reinterpret_cast<const Slice*>(src);
         auto* dst_slice = reinterpret_cast<Slice*>(dst->mutable_cell_ptr());
 
+        // Because of history bug, we ingest some NULL HLL data in storage,
+        // which slice data is nullptr or invalid address,
+        // we must handle this case to avoid process crash.
+        HyperLogLog* hll = nullptr;
```

Comment:
I don't agree simply handle null here. If we do this way, we must handle null forever, which is terrible.

At least，We should find a way to convert the invalid null HLL to empty and Remove null handle in future version.

Reply:
I think this pr is a fix for #1908, which handles the scenario where HLL data is nullptr but does not handle dirty data. And according to it's comments,  it suggested to remove these check for null HLL column later.

Reply:
If the data has dirty, I think we should repair it. 
You could reload the table data or develop a tool to repair dirty data. 
I think we shouldn't make code to be compatible with dirty data. which means we forever couldn't remove the compatible code.